uw-frame-components/css/buckyless/notifications.less

```diff
@@ -133,6 +133,22 @@
   font-size: 20px;
 }
 .menu-notification-link {
+  .arrow-down {
+  	width: 0; 
+  	height: 0; 
+  	border-left: 7px solid transparent;
+  	border-right: 7px solid transparent;
+  	border-top: 7px solid lighten(@color1, 20%);
+    z-index:2000;
```

Comment:
Yea, it's necessary to place it over the app-header. Even if it's 1000-something it doesn't show. z-index goes up to 9999, so I don't think it's too big of a deal...I'm not sure if there is a best practice for dealing with z-indexes.